ads/rakutenunifiedads.md

```diff
@@ -0,0 +1,37 @@
+<!---
+Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Rakuten Unified Ads
+
+## Example
+
+```html
+<amp-ad width="300" height="250" type="rakutenunifiedads" data-id="63">
+</amp-ad>
+```
+
+## Configuration
+
+For details on the configuration semantics, please contact the [ad network](#configuration) or refer to their documentation.
```

Comment:
Maybe you want to substitute this row with links to your website?

Reply:
Actually, we don't have a website to explain it, so I will delete this line.